### ISSUE TITLE
Add Collaboration

### ISSUE THREAD

**Issue by odiplol** [haven1433/HexManiacAdvance#214]
It would probably be difficult to implement, but i'd love to work together on my rom hack with my friends.

**Comment by odiplol** [comment]
It would be difficult to implement, but what about live collaboration, where your friend can edit the map and do everything in real time along side you, just a thought.

**Comment by haven1433** [comment]
Yeah live collaboration is definitely out, sharing a live session like that across computers is something I don't know how to do
